### DISCUSSION TITLE
Email error

### DISCUSSION THREAD

**Discussion by ssiddiqui87** [sct/overseerr#582] [Help] [answered]
I can't seem to get my emails working. I am using Mailcow in docker and I tried using the same settings I use in Ombi for email notifications however the settings that work in Ombi don't seem to work in Overseer. Checking the logs, I see the following error when trying to test:

[error][Notifications]: Hostname/IP does not match certificate's altnames: Host: mailcowdockerized_postfix-mailcow_1. is not in the cert's altnames: DNS:my.domain.name

In Ombi I have "Disable Certificate Checking" enabled and I think I would need something similar in Overseer to get emails working. I have "allow self signed certificates" checked in Overseer

Anyone have any ideas on how I can get this working?

**Comment by sct** [comment, marked as answer]
Try setting the environment variable `NODE_TLS_REJECT_UNAUTHORIZED` to `0`

**Reply by ssiddiqui87** [reply]
That worked, thank you!
